test/functional/test_framework/messages.py

```diff
@@ -615,7 +615,7 @@ def __init__(self, header=None):
             self.calc_sha256()
 
     def set_null(self):
-        self.nVersion = 1
+        self.nVersion = 4
```

Comment:
fac90c55be478f0323eafa1d560e it seems that using `from test_framework.blocktools import VERSIONBITS_LAST_OLD_BLOCK_VERSION` in `messages.py` would be a circular import, but perhaps
```suggestion
        self.nVersion = 4  # VERSIONBITS_LAST_OLD_BLOCK_VERSION in test_framework.blocktools
```

Reply:
Might do if I have to re-touch